posts/ja/2023-01-30-22.0.0.11_ja.adoc

```diff
@@ -0,0 +1,467 @@
+---
+layout: post
+title: "Java SE 19 and distributed security cache support in Open Liberty 22.0.0.11"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/mbroz2
+author_github: https://github.com/mbroz2
+seo-title: Open Liberty 22.0.0.11 での Java SE 19 および分散セキュリティー・キャッシュのサポート - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.11 provides support for Java SE 19 and distributed security caches which can improve performance and failure recovery.  Also included are improvements to the BELL feature along with CVE and bug fixes.
+blog_description: Open Liberty 22.0.0.11 provides support for Java SE 19 and distributed security caches which can improve performance and failure recovery.  Also included are improvements to the BELL feature along with CVE and bug fixes.
```

Comment:
`seo-description` and `blog_description` need to be translated from `en` to `ja`

Reply:
Hi @una-tapa, I have used machine translation for the suggested change below. Can you help me double check if the translation is appropriate?
```suggestion
seo-description: Open Liberty 22.0.0.11 は、 Java SE 19 および分散セキュリティー・キャッシュをサポートします。これにより、パフォーマンスと障害のリカバリーが改善されます。  また、 CVE およびバグ修正と共に BELL フィーチャーの改善も含まれています。
blog_description: Open Liberty 22.0.0.11 は、 Java SE 19 および分散セキュリティー・キャッシュをサポートします。これにより、パフォーマンスと障害のリカバリーが改善されます。  また、 CVE およびバグ修正と共に BELL フィーチャーの改善も含まれています。
```